Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -423,6 +427,19 @@ extension HTTPClient.Configuration {
             self.read = read
         }
     }
+
+    /// Specifies redirect processing settings.
+    public enum RedirectPolicy {
+        /// Redirects are not followed.
+        case disallow
+        /// Redirects are followed with a specified limit. Cycle detection reqiures that all visited URL's are kept in memory.
+        case allow(max: Int, allowCycles: Bool)
```

Comment:
not: "disabled" and "enabled" seem more natural than "disallow" and "allow"? since this is not some security policy?

Reply:
this struct was renamed to `Policy`, do you think `RedirectPolicy.enable` is better than `RedirectPolicy.allow`? I don't have a strong opinion here, wdyt? cc @ianpartridge

Reply:
In the tests it reads as `redirects: .allow(max: 5, allowCycles: false)` which seems fine (although I would prefer `redirection:` instead of `redirects:`). `.allow` makes sense to me because redirection is something that is requested by the remote HTTP server via an HTTP return code, and we decide whether to allow it or not.

Reply:
How about `RedirectConfiguration.disallow` and `RedirectConfiguration.follow`?

Reply:
I like `.follow` 🙂